modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -1182,6 +1182,15 @@ their dependencies published before enabling the `printArtifactPublishCommands`
 task.
 - [LPS-71005]: Update the [Liferay Gradle Plugins] dependency to version 3.2.15.
 
+## 3.2.5 - 2017-03-08
+
+### Added
+- [LPS-63943]: Print out artifacts publish commands file path.
```

Comment:
The `writeArtifactsPublishCommands` task creates a `.sh` file with a list of commands that Brian runs in order to publish modules. I added a log, so at the end of the task it writes where this `.sh` file is.